### ISSUE TITLE
Incorrect unused-import message

### ISSUE THREAD

**Issue by andreymal** [PyCQA/pylint#2928]
```python
def test1(s):
    from six import PY2, text_type
    return s.encode("utf-8") if PY2 else s
```

### Current behavior

```
pylint_wat.py:2:4: W0611: Unused PY2 imported from six (unused-import)
```

### Expected behavior

```
pylint_wat.py:2:4: W0611: Unused text_type imported from six (unused-import)
```

### pylint --version output

```
pylint 2.3.1
astroid 2.2.5
Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0]
```

**Comment by PCManticore** [comment]
Thank you, I can reproduce the issue.
